showcase/src/elm/Utils/FileParser.elm

```diff
@@ -0,0 +1,389 @@
+module Utils.FileParser exposing (elliefy)
+
+
+{- | File-related Logic
+-}
+
+import Parser exposing ((|.), (|=), DeadEnd, Parser, Problem(..), Step(..))
+import Url.Builder
+
+
+
+-------------------
+-- PARSING STUFF
+-- Elliefy makes the code renderable in Ellie. WIP
+-------------------
+
+
+elliefy : String -> { title : String, code : String }
```

Comment:
As discussed in my [other comment](https://github.com/supermacro/elm-antd/pull/64#discussion_r515130123), I think a `Result` should already be returned here.

You can add as many `Debug.log` calls as you need. Removing the `Debug.log` calls can be done as a final commit to this PR once things are good to merge.